packages/sdk-core/src/model/report/BacktraceReport.ts

```diff
@@ -1,7 +1,60 @@
 import { BacktraceAttachment } from './BacktraceAttachment';
+import { BacktraceErrorType } from './BacktraceErrorType';
 
-export interface BacktraceReport {
-    readonly attributes: Record<string, unknown>;
-    readonly attachments: BacktraceAttachment[];
-    toData(): Record<string, unknown>;
+export class BacktraceReport {
+    /**
+     * Report classifiers
+     */
+    public readonly classifiers: string[] = [];
+    /**
+     * Report annotations
+     */
+    public readonly annotations: Record<string, object> = {};
+    /**
+     * Report stack trace
+     */
+    public readonly stackTrace: string;
+
+    /**
+     * Report message
+     */
+    public readonly message: string;
+
+    /**
+     * Report inner errors
+     */
+    public readonly innerReport: unknown[] = [];
+
+    /**
+     * Number of frames generated by the Backtrace SDK
+     */
```

Comment:
The doc is invalid.

Reply:
Adjusted the comment based on our conversation